lib/segment/src/types.rs

```diff
@@ -1350,7 +1350,7 @@ pub struct GeoPolygon {
     /// Interior lines (if present) bound holes within the surface
     /// each GeoLineString must consist of a minimum of 4 points, and the first
     /// and last points must be the same.
-    pub interiors: Vec<GeoLineString>,
+    pub interiors: Option<Vec<GeoLineString>>,
```

Comment:
here is the important change.

Scrolling above you will find the shadow type which already had the optional field

```rust
#[derive(Deserialize)]
pub struct GeoPolygonShadow {
    pub exterior: GeoLineString,
    pub interiors: Option<Vec<GeoLineString>>,
}
```

Reply:
Thanks for capturing!